edx-platform/pearson-ironwood/lms/templates/static_templates/server-error.html

```diff
@@ -0,0 +1,36 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%inherit file="../main.html" />
+
+<main id="main" aria-label="Content" tabindex="-1">
+    <section class="outside-app">
+        <h1>
+            <%block name="pageheader">
+                % if page_header:
+                    ${page_header}
+                % else:
+                    ${Text(_(u"There has been a 500 error on the {platform_name} servers")).format(
+                    platform_name=HTML("<em>{platform_name}</em>").format(platform_name=Text(static.get_platform_name()))
+                    )}
+                % endif
+            </%block>
+        </h1>
+        <p>
+            <%block name="pagecontent">
+                % if page_content:
+                    ${page_content}
+                % else:
+                    ${Text(_('Please wait a few seconds and then reload the page. If the problem persists, please visit Support {support_link}.')).format(
+                    support_link=HTML('<a href="{support_link}">Support</a>').format(
```

Comment:
Wrong indentation. @luismorenolopera 
Not sure if you should mark "Support" for extraction, can you confirm it?
Could you change the first support link variable to: support link_tag?

Reply:
> But then you have to go to each template and inherit from that common folder, am I wrong? @luismorenolopera

yes, it is what would happen

Reply:
yes, support must be marked